Makefile

```diff
@@ -98,16 +98,19 @@ base-node-image:
 	docker buildx build --platform=linux/amd64 -t eu.gcr.io/stashify-218417/basenode:latest . -f Dockerfile.basenode
 	docker push eu.gcr.io/stashify-218417/basenode:latest
 
-build-search-images: build-local-node build-local-sidecar
+build-search-images: build-local-node build-sidecar
```

Comment:
where is the target `build-local-node`?

Reply:
Oh, it's not there! Good catch! Do you think it was the current `build-node`?

Reply:
I just removed the `build-search-images` target